app/2.0/docs/tools/publish.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Publish your elements
+---
+
+<!-- toc -->
+
+You can publish your elements on WebComponents.org to make them available on the Internet; or you can host them privately.
+
+## Publish your elements publicly on WebComponents.org
+
+The easiest way to make your reusable elements available is to [publish your elements on WebComponents.org](https://www.webcomponents.org/publish).
+
+Before you publish on WebComponents.org, create a GitHub repository for your element. Make sure your repository has the following:
+
+* An [open source license](https://opensource.org/licenses/alphabetical). TODO how does the user communicate their license? Do they need a license file? Is this automated? Etc
```

Comment:
@arthurevans hoping you can help with the TODO

Reply:
It usually involves adding a license comment to the top of your source files. People often also put a copy in a LICENSE.txt or LICENSE.md file to make it easy for people to see when they look at the repo. See also:

https://opensource.org/faq#apply-license